roles/calico/templates/calico.service.j2

```diff
@@ -10,18 +10,22 @@ ExecStart=/usr/bin/docker run --net=host --privileged \
  --name=calico-node \
  -e WAIT_FOR_DATASTORE=true \
  -e FELIX_DEFAULTENDPOINTTOHOSTACTION=ACCEPT \
- -e CALICO_IPV4POOL_IPIP=always \
- -e FELIX_IPV6SUPPORT=false \
+ -e CALICO_IPV4POOL_IPIP={{ calico_ipv4pool_ipip }} \
+ {% if calico_ipv4pool_cidr is defined %}
```

Comment:
Isn't `calico_ipv4pool_cidr` always defined since it's declared as a default var? Shouldn't we just nix the `{% if calico_ipv4pool_cidr is defined %}` and always set `-e CALICO_IPV4POOL_CIDR={{ calico_ipv4pool_cidr }}`?